extensions/wikia/AdEngine/js/provider/factoryWikiaGpt.js

```diff
@@ -77,8 +63,12 @@ define('ext.wikia.adEngine.provider.factory.wikiaGpt', [
 				lookups.extendSlotTargeting(slotName, slotTargeting);
 			}
 
-			gptHelper.pushAd(slotName, slotPath, slotTargeting, doSuccess, doHop);
-			flushIfNeeded();
+			if (gptSraHelper && extra.sraEnabled) {
```

Comment:
Let's then fail visibly when one requests SRA but the module is not loaded (rather than just not using SRA).